rallyrolebot/validation.py

```diff
@@ -59,13 +60,31 @@ async def is_valid_channel(ctx, channel_name):
     return True
 
 
-async def is_valid_coin(ctx, coin_name):
-    """
-        Poll the rally API to determine wheter or not the coin_name exists
-        TODO: Use discord.py converters instead of is_valid_coin check
-    """
-    valid = rally_api.valid_coin_symbol(coin_name)
+async def is_valid_creator_coin(coin_symbol):
+    valid = valid_coin_symbol(coin_symbol)
+    if not valid:
+        return False
+    return True
+
+async def is_valid_common_coin(coin_symbol):
+    valid = valid_coin(coin_symbol)
     if not valid:
-        await ctx.send("Coin " + coin_name + " does not seem to exist")
         return False
     return True
```

Comment:
A bit of my fault here, validation.py is on its way out (to be replaced by ```utils.checks```) - please create discord [Advanced Converter](https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html#converters) classes in ```utils.converters``` named CommonCoin and CreatorCoin. raise an ```errors.InvalidCoin``` exception if not valid. 

Apologies! There will be more documentation on the discord.py way of doing things soon.